elliottlib/cli/__main__.py

```diff
@@ -392,6 +392,62 @@ def find_cve_trackers(runtime, cve, status):
     for b in bug_list:
         click.echo("{}\t{:15s}\t{}".format(b.bug_id, b.status, b.summary))
 
+
+def _get_advisory_nvrs(advisory):
+    try:
+        green_prefix("Fetching advisory builds: ")
+        click.echo("Advisory - {}".format(advisory))
+        builds = elliottlib.errata.get_builds(advisory)
+    except GSSError:
+        exit_unauthenticated()
+    except elliottlib.exceptions.ErrataToolError as ex:
+        raise ElliottFatalError(getattr(ex, 'message', repr(ex)))
+
+    all_advisory_nvrs = {}
+    # Results come back with top level keys which are brew tags
+    green_prefix("Looping over tags: ")
+    click.echo("{} tags to check".format(len(builds)))
+    for tag in builds.keys():
+        # Each top level has a key 'builds' which is a list of dicts
+        green_prefix("Looping over builds in tag: ")
+        click.echo("{} with {} builds".format(tag, len(builds[tag]['builds'])))
+        for build in builds[tag]['builds']:
+            # Each dict has a top level key which might be the actual
+            # 'nvr' but I don't have enough data to know for sure
+            # yet. Also I don't know when there might be more than one
+            # key in the build dict. We'll loop over it to be sure.
+            for name in build.keys():
+                n, v, r = name.rsplit('-', 2)
+                version_release = "{}-{}".format(v, r)
+                all_advisory_nvrs[n] = version_release
+
+    return all_advisory_nvrs
+
+
+@cli.command("get-golang-versions", short_help="Get version of Go used for builds attached to an advisory")
+@click.argument('advisory', type=int)
+@pass_runtime
+def get_golang_versions(runtime, advisory):
+    """
+    Only works for RPM builds.
+
+    Usage:
+\b
+    $ elliott --group openshift-3.7 get-golang-versions ID
+"""
+    all_advisory_nvrs = _get_advisory_nvrs(advisory)
+
+    click.echo("Found {} builds".format(len(all_advisory_nvrs)))
+    for package, value in all_advisory_nvrs.items():
```

Comment:
We prefer to use the term `component` rather than `package` to reduce confusion. Could you rename it to `component` or `name`?

Reply:
Done